### ISSUE TITLE
Always display byline authors in citation_author metadata

### ISSUE THREAD

**Issue by gabestein** [pubpub/pubpub#2352]
### What went wrong, step-by-step?
1. Visited a Pub with byline authors who have roles
2. These authors did not show up in citation_author metadata.

### What did you expect to happen?
Have them show up. This is required for Google Scholar inclusion.

### What URL can we use to see the issue?
https://research.arcadiascience.com/pub/result-comparative-crawling/draft

### What browser/operating system were you using?

===

**Comment by qweliant** [comment]
are you saying `const contributorRoles = ['Writing – Review & Editing', 'Editor', 'Series Editor'];` should not exist anymore?

**Comment by gabestein** [comment]
Yes, but only for the purposes of html metadata -- it still needs to work that way for citations, pdfs, attributions, etc.
